atlas

```diff
@@ -407,17 +425,18 @@ if [ $# -gt 0 ]; then
 	case $cmd in
 		pull)
 			parse_pull "$@"
-      set_pull_params
-
-      if [ -z "$SILENT" ];
-      then
-        display_pull_params
-      fi
-      # allow mocking pull_translations
-      __ begin_pull_translations_mock __
-      pull_translations
-      __ end_pull_translations_mock __
-      ;;
+            eval "set -- $PULL_REST"
```

Comment:
If I'm not mistaken, this is a must to ensure we get the remaining positional arguments.